image/templates/helm/stackrox-central/config/centraldb/postgresql.conf.default

```diff
@@ -64,7 +64,7 @@ listen_addresses = '*'              	# StackRox customized
 					# defaults to 'localhost'; use '*' for all
 					# (change requires restart)
 #port = 5432				# (change requires restart)
-max_connections = 100			# (change requires restart)
+max_connections = 200			# (change requires restart)
```

Comment:
Do we need to share this with ACSCS?

Reply:
my read is that rds uses a different algorithm

Reply:
What error would we see in case of over this limit?

Reply:
too many clients connected